contracts/Liquidation.sol

```diff
@@ -334,14 +335,13 @@ contract Liquidation is ILiquidation, Ownable {
         require(amount > 0, "LIQ: Liquidation amount <= 0");
 
         /* Liquidated account's balance */
-        Types.AccountBalance memory liquidatedBalance =
-            tracer.getBalance(account);
+        Balances.Account memory liquidatedBalance = tracer.getBalance(account);
 
         uint256 amountToEscrow =
             verifyAndSubmitLiquidation(
-                liquidatedBalance.quote,
+                liquidatedBalance.position.quote,
```

Comment:
Are you planning to modify all these calls that separate out base and quote into arguments so that they simply take in a position? Would be worthwhile in a new PR imo

Reply:
@raymogg Yes to both statements.